### ISSUE TITLE
Can't delete sub-directories

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#124]
```
What steps will reproduce the problem?
1. Import a directory structure into a new repository
       Example: Foobar        (Top level directory)
                    Foo       (Sub directory of Foobar)
                    Bar       (Sub directory of Foobar)

2. Click the 'Delete' button in the svnX Repository window.

3. Select the top level directory (Foobar from above example).

What is the expected output? What do you see instead?

   I expect the top level directory (Foobar) to highlight, and see the sub-directories (Foo and Bar) in the next column.
   Instead, nothing happens at all. As a result, I am unable to delete any of the sub-directories since they are not displayed.

What version of the product are you using? On what operating system?

   svnX version 1.3
   Mac OS X 10.6.5

Please provide any additional information below.

   This is only an issue if there is only one top level directory (If you click on 'root', there is only one directory in the column to the right (Foobar)).

   Workaround: Use the 'Make Dir' button to add a second top level directory. As strange as this sounds, it solves the problem.

```

Original issue reported on code.google.com by `ian.ur...@gmail.com` on 24 Nov 2010 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
This is fixed and will appear in the next release.
```

Original comment by `chris...@gmail.com` on 14 Apr 2011 at 1:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Issue 135 has been merged into this issue.
```

Original comment by `chris...@gmail.com` on 21 Apr 2011 at 5:40

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 1.3.1.
```

Original comment by `chris...@gmail.com` on 1 Aug 2011 at 2:47
- Changed state: **Fixed**
